From 28ff72ce58183c2d9c910badf5b94de614cf2c5e Mon Sep 17 00:00:00 2001
From: TheAtt1la <alittauwu@gmail.com>
Date: Wed, 1 Mar 2023 23:39:45 +0100
Subject: [PATCH 2/2] fix build name

---
 config/version.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/version.mk b/config/version.mk
index d68d3c44..f8fbccbd 100644
--- a/config/version.mk
+++ b/config/version.mk
@@ -8,5 +8,5 @@ CUSTOM_BUILD_DATE := $(CUSTOM_DATE_YEAR)$(CUSTOM_DATE_MONTH)$(CUSTOM_DATE_DAY)-$
 
 CUSTOM_PLATFORM_VERSION := 13.0
 
-CUSTOM_VERSION := PixelOS_$(CUSTOM_BUILD)-$(CUSTOM_PLATFORM_VERSION)-$(CUSTOM_BUILD_DATE)
+CUSTOM_VERSION := PixelOS_gsi-$(CUSTOM_PLATFORM_VERSION)-$(CUSTOM_BUILD_DATE)
 CUSTOM_VERSION_PROP := thirteen
-- 
2.34.1

